### PR TITLE
Issue #149: Get listings for a specific user

### PR DESCRIPTION
**Pull request by RonnieDsouza**
**Changelog**
- Created 2 GET routes for:
    -  all listings for a specific user 
    - specific listing for a specific user. 
- Front end also receives userid upon successful login.
- Added DROP TABLE at the beginning of init.sql so both tables are dropped and recreated to ensure they rebuild with any updates.

**Screenshots**
1. All listings for a specific user. 
User 1->Listing 1, 2. 
User 2-> Listing 3. 
Both listings for a user are sent. 
image

2. Requesting a specific user's specific listing. 
The correct Listing is sent. 
image

When a listing and user don't match, nothing is sent. 
image

3. UserID is sent upon successful Login
image

### CODE REVIEW

**Review by Mik3Qiao — APPROVED** [review]

**Review by AzizulHakim96 — APPROVED** [review]

File: server/controllers/user.js [hunk 1]
```diff
@@ -53,14 +54,19 @@ class UserController {
 
   authenticateUser = async (req, res, next) => {
     try {
+      // console.log(req);
```
Comment (jcsnorlax97) [inline server/controllers/user.js:57]: picky: can remove this unused code.

File: server/controllers/user.js [hunk 2]
```diff
@@ -53,14 +54,19 @@ class UserController {
...
       const user = await this.userService.getUserViaEmail(req.body.email);
+      // const userId;
```
Comment (jcsnorlax97) [inline server/controllers/user.js:59]: picky: can remove this unused code.

File: server/controllers/user.js [hunk 3]
```diff
@@ -53,14 +54,19 @@ class UserController {
...
       if (!user) {
         console.log('No such user');
         next(ApiError.unauthenticated('Please check your login info.'));
       } else if (await bcrypt.compare(req.body.password, user.password)) {
+        // userId = user.userId;
+        console.log(user.id);
+
```
Comment (jcsnorlax97) [inline server/controllers/user.js:66]: picky: can remove this unnecessary code.

**Review by jcsnorlax97 — APPROVED** [review]
In general it looks good to me! Good job, Ronnie!

File: server/controllers/user.js (hunk 2 above)
Comment (RonnieDsouza) [reply server/controllers/user.js:59]: Sorry! Didn't see these!

**Pull request merged**